<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReflRunsTabWidget</class>
 <widget class="QWidget" name="ReflRunsTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RunsTab</string>
  </property>
   <layout class="QVBoxLayout" name="layoutMain">
    <property name="margin">
     <number>1</number>
    </property>
     <item>
       <widget class="QMenuBar" name="menuBar">
         <property name="geometry">
           <rect>
             <x>0</x>
             <y>0</y>
             <width>959</width>
             <height>18</height>
           </rect>
         </property>
         <property name="nativeMenuBar">
           <bool>false</bool>
         </property>
         <widget class="QMenu" name="menuTable">
           <property name="title">
             <string>&amp;Reflectometry</string>
           </property>
         </widget>
         <widget class="QMenu" name="menuRows">
           <property name="title">
             <string>&amp;Edit</string>
           </property>
         </widget>
         <addaction name="menuTable"/>
         <addaction name="menuRows"/>
       </widget>
     </item>
    <item>
     <widget class="QSplitter" name="splitterTables">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
       <property name="styleSheet">
        <string>QSplitter::handle { background-color:white }</string>
       </property>
       <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QGroupBox" name="groupSearchPane">
       <property name="title">
        <string>Search Runs</string>
       </property>
       <layout class="QVBoxLayout" name="layoutSearchPane">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <layout class="QFormLayout" name="layoutSearchForm">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelInstrument">
            <property name="text">
             <string>Instrument:</string>
            </property>
            <property name="buddy">
             <cstring>comboSearchInstrument</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboSearchInstrument">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Select the instrument to search with</string>
            </property>
            <property name="whatsThis">
             <string>Specifies which instrument you're searching for data from.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelInvestigationId">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Investigation Id:</string>
            </property>
            <property name="buddy">
             <cstring>textSearch</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="textSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Investigation to search for</string>
            </property>
            <property name="whatsThis">
             <string>Specifies the investigation id that you are searching for runs from.</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="buttonSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Search</string>
            </property>
            <property name="whatsThis">
             <string>Searches ICAT for runs from the given instrument with the given investigation id.</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QComboBox" name="comboTransferMethod">
            <property name="whatsThis">
             <string>The strategy for searching and transfering files. See help.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableSearchResults">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>60</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutSearchBottomRow">
          <item>
            <widget class="QProgressBar" name="progressBar">
              <property name="whatsThis">
                <string>Shows the current progress when transferring runs.</string>
              </property>
              <property name="value">
                <number>0</number>
              </property>
            </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonTransfer">
            <property name="text">
             <string>Transfer</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupProcessPane">
         <property name="title">
           <string>Process Runs</string>
         </property>
         <layout class="QVBoxLayout" name="layoutProcessPane">
           <property name="spacing">
             <number>1</number>
           </property>
           <property name="margin">
             <number>1</number>
           </property>
           <item>
             <widget class="QToolBox" name="toolbox">
             </widget>
           </item>
         </layout>
       </widget>
       </widget>
    </item>
   </layout>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../../../../MantidPlot/icons/icons.qrc">
     <normaloff>:/folder.png</normaloff>:/folder.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionTransfer">
   <property name="icon">
    <iconset resource="../../../../MantidPlot/icons/icons.qrc">
     <normaloff>:/append_drag_curves.png</normaloff>:/append_drag_curves.png</iconset>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
   <property name="toolTip">
    <string>Transfer the selected run(s) to the processing table.</string>
   </property>
   <property name="whatsThis">
    <string>Transfers the selected runs into the processing table.</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>comboSearchInstrument</tabstop>
  <tabstop>tableSearchResults</tabstop>
 </tabstops>
  <customwidgets/>
  <resources>
  <include location="../../../../MantidPlot/icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonSearch</sender>
   <signal>clicked()</signal>
   <receiver>actionSearch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textSearch</sender>
   <signal>returnPressed()</signal>
   <receiver>actionSearch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
